core/translate.c

```diff
@@ -107,6 +107,8 @@ translate_walk_init(translate_walk_t *walk, byte *start_cache, byte *end_cache,
     walk->mc = mc;
     walk->start_cache = start_cache;
     walk->end_cache = end_cache;
+    for (int r = 0; r < REG_SPILL_NUM; r++)
+        walk->reg_spilled[r] = UINT_MAX;
```

Comment:
A comment above and perhaps here too would help to understand what UINT_MAX is supposed to mean

Reply:
Added comment.